api/models.py

```diff
@@ -1377,11 +1376,13 @@ class ActionOrg:
     NATIONAL_SOCIETY = 'NTLS'
     FOREIGN_SOCIETY = 'PNS'
     FEDERATION = 'FDRN'
+    GOVERNMENT = 'GOV'
 
     CHOICES = (
         (NATIONAL_SOCIETY, _('National Society')),
         (FOREIGN_SOCIETY, _('Foreign Society')),
         (FEDERATION, _('Federation')),
+        (GOVERNMENT, _('Government')),
```

Comment:
Do we need this, for `informal_update`?

Reply:
We have one more option Government in the mockup, so it is added.